<?xml version="1.0" ?>
<launch>
    <arg name="robot_ip"/>
    <arg name="arm_id"/>
    <arg name="joint_goal0"/>
    <arg name="joint_goal1"/>
    <arg name="joint_goal2"/>
    <arg name="joint_goal3"/>
    <arg name="joint_goal4"/>
    <arg name="joint_goal5"/>
    <arg name="joint_goal6"/>

    <group ns="$(arg arm_id)">
      <!-- (start your robot's MoveIt! stack, e.g. include its moveit_planning_execution.launch) -->
      <include file="$(find panda_moveit_config)/launch/panda_control_moveit_rviz.launch">
         <arg name="robot_ip" value="$(arg robot_ip)"/> <!-- set your robot ip -->
         <arg name="load_gripper" value="true"/>
         <arg name="launch_rviz" value="true"/>
      </include>  

      <param name="arm_id" value="$(arg arm_id)"/>

      <param name="joint_goal0" value="$(arg joint_goal0)"/>
      <param name="joint_goal1" value="$(arg joint_goal1)"/>
      <param name="joint_goal2" value="$(arg joint_goal2)"/>
      <param name="joint_goal3" value="$(arg joint_goal3)"/>
      <param name="joint_goal4" value="$(arg joint_goal4)"/>
      <param name="joint_goal5" value="$(arg joint_goal5)"/>
      <param name="joint_goal6" value="$(arg joint_goal6)"/>

      <node name="move_group_python_interface_tutorial" pkg="franka_example_controllers" type="move_group_python_interface.py" output="screen" required="true" />
     </group>

</launch>
